rudder-by-example/src/application/install-exe-or-msi.adoc

```diff
@@ -0,0 +1,78 @@
+= Install a software from a .exe or .msi binary
```

Comment:
A program ("software" is uncountable)

I would also replace "binary" by "installer".